system_tests/common_test.go

```diff
@@ -243,12 +244,13 @@ func CreateTestNodeOnL1(
 	t *testing.T,
 	ctx context.Context,
 	isSequencer bool,
+	feedErrChan chan error,
```

Comment:
Instead of passing the feedErrChan around everywhere, maybe `CreateTestNode*` could start a background goroutine that selects on `feedErrChan` and context cancellation, and fails the test if `feedErrChan` gets an error.

Reply:
My thinking was that we could call `t.Fail` in a background goroutine which reads from `feedErrChan`. It's a bit inconvenient to need to pass around `feedErrChan` as-is.